Makefile.toml

```diff
@@ -89,6 +89,11 @@ BUILDSYS_UPSTREAM_SOURCE_FALLBACK = "false"
 # if the license check fails.
 BUILDSYS_ALLOW_FAILED_LICENSE_CHECK = "false"
 
+# We require package licenses to be cached to build an image. If you're working on a
+# local change and don't have license information yet, you can run with `-e
+# BUILDSYS_ALLOW_FAILED_LICENSE_FETCH=true` to allow the build to continue even
+BUILDSYS_ALLOW_FAILED_LICENSE_FETCH = "false"
```

Comment:
I don't think we'd want to continue if we try to retrieve a license for which we have a remote URL and fail - that'd just turn into a package build failure later, and the situation described in the comment here is easily fixed by amending `Licenses.toml` to drop the remote URL.

We're really more concerned about "unexpected" network fetches, where the data is supposed to be local but isn't for some reason. So I'd probably make this a `BUILDSYS_UPSTREAM_LICENSE_FETCH` boolean that gates whether we reach out to the different URLs - it's a source of nondeterminism for builds and should be opt-in.